cfme/common/provider.py

```diff
@@ -426,7 +434,7 @@ def load_details(self, refresh=False):
         if not self._on_detail_page():
             logger.debug("load_details: not on details already, navigating")
             sel.force_navigate('{}_{}'.format(self.page_name, self.detail_page_suffix),
-                context={'provider': self})
```

Comment:
They weren't deleted but pushed a tab forward. I had PyCharm fix that for me that way for some reason. It would shout at me the other way